### PR TITLE
automate load config file from default path

### LINKED ISSUE #128: crictl should automate load config file from default path.
crictl should automate load config file from default path  no matter wether it is exist in command line args.
Now even though I has writed config file in /etc/crictl.yaml , I must enter "crictl  --config /etc/crictl.yaml".

@Random-Liu

### PR DESCRIPTION
**Pull request by yanxuean**
fix #128 
Signed-off-by: yanxuean <yan.xuean@zte.com.cn>

### CODE REVIEW

**Comment by yanxuean** [comment]
@feiskyer    @Random-Liu   PTAL

**Comment by feiskyer** [comment]
@yanxuean Thanks for the PR. crictl will always fatal if `/etc/crictl.yaml` doesn't exist, while this is not expected. 

Could you update the PR and fallback to flags and env if `/etc/crictl.yaml` doesn't exist?

**Comment by yanxuean** [comment]
yes，I will solve  it

**Comment by yanxuean** [comment]
@feiskyer  @Random-Liu  PTAL, Thanks

File: cmd/crictl/main.go [hunk 1]
```diff
@@ -133,8 +134,26 @@ func main() {
 	}
 
 	app.Before = func(context *cli.Context) error {
+		isUseConfig := false
 		configFile := context.GlobalString("config")
-		if configFile == "" {
+		if !context.IsSet("config") {
```
Comment (feiskyer) [inline cmd/crictl/main.go]: This could be simplified as

```go
if _, err := os.Stat(configFile); err == nil {
    isUseConfig = true
}
```

Comment (yanxuean) [reply cmd/crictl/main.go]: done

Comment (yanxuean) [reply cmd/crictl/main.go]: We can not omit the error handling. 
We always show result to user when user self indicate the config file in command line.  
But we should not show result to user when the default config file is absence. This is a normal case.

**Comment by feiskyer** [comment]
@yanxuean  LGTM. thanks a lot

**Comment by Random-Liu** [comment]
@yanxuean Thanks!

**Pull request merged**